examples/diffusion/python_controlnet_canny_sd_15/requirements.txt

```
@@ -21,13 +21,10 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #####################################################################################
--f https://repo.radeon.com/rocm/manylinux/rocm-rel-6.0/
-torch==2.1.1
+--extra-index-url https://test.pypi.org/simple
```

```suggestion
-r torch_requirements.txt
--extra-index-url https://test.pypi.org/simple
```
See above comment, I think this should work